drivers/wifi/infineon/airoc_whd_hal_common.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2023 Cypress Semiconductor Corporation (an Infineon company) or
+ * an affiliate of Cypress Semiconductor Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "airoc_whd_hal_common.h"
+#include "airoc_wifi.h"
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/gpio.h>
+
+LOG_MODULE_DECLARE(infineon_airoc_wifi, CONFIG_WIFI_LOG_LEVEL);
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/******************************************************
+ *                 Function
```

Comment:
nit: Is this required? It seems a bit vestigial.

Reply:
Hmm ... true;  that also came from the original source file, and probably ought to be removed.